src/Schema/Scripts/PostDeployment.sql

```diff
@@ -1,8 +1,11 @@
-﻿/* 
+﻿/*
  Post-Deployment Script base endpoint.
  This file contains SQL statements that will be appended to the build script.
  Scripts will run according to th order in this file!
 */
 
 :r ..\StaticData\static_Roles.sql
-:r ..\StaticData\static_UsersLogin.sql
\ No newline at end of file
+:r ..\StaticData\static_UsersLogin.sql
```

Comment:
This file is more convenient to edit during merging process, when you pull changes from master.